guides/common/modules/proc_associating-partition-tables-with-disk-encryption.adoc

```diff
@@ -0,0 +1,29 @@
+[id="associating-partition-tables-with-disk-encryption_{context}"]
+= Associating partition tables with disk encryption
+
+{Project} contains partition tables that encrypt the disk of your host using Linux Unified Key Setup (LUKS) during host provisioning.
+Optionally, you can also bind the disk to a Tang server through Clevis for decryption during boot.
+You must associate the partition table with your operating system entry.
+Then, you assign the partition table to your host group or select it manually during provisioning.
+
+Encrypted disks on hosts protect data at rest.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* > *Provisioning Setup* > *Operating Systems*.
+. Select your {client-os} entry.
+. On the *Partition Table* tab, associate the `{client-provisioning-template-type} default encrypted` to your operating system entry.
+For more information, see xref:creating-partition-tables_{context}[].
+. Create a host group that uses the `{client-provisioning-template-type} default encrypted` partition table.
+For more information, see {ManagingHostsDocURL}Creating_a_Host_Group_managing-hosts[Creating a host group] in _{ManagingHostsDocTitle}_.
+. Decrypt the disk of your host during boot time by using one of the following options:
++
+* LUKS encryption: Add the host parameter `disk_enc_passphrase` as type *string* and your cleartext passphrase of the LUKS container as the value.
```

Comment:
```suggestion
* LUKS encryption: Add the host `disk_enc_passphrase` parameter as type *string* and your cleartext passphrase of the LUKS container as the value.
```
Right? Applicable also for the line below.

Reply:
IMO, it's a lot of words for a user to scan. I think Max's original form, with added commas, would make it more readable:

"Add the host parameter, `disk_enc_passphrase`, as type *string* and your cleartext passphrase of the LUKS container as its value."

I think the commas make it easier to parse the meaning because the user sees that "host parameter" =  `disk_enc_passphrase`. Without the punctuation, it's not immediately apparent that 'host parameter' is a compound noun, with `disk_enc_passphrase` in apposition. 

Your suggestion, "host `disk_enc_passphrase` parameter", is grammatically correct. But I think that multiple adjectives  (host `disk_enc_passphrase`) modifying a single noun require more effort to understand. At least, my brain needs to work a little harder. :-)